packages/live-share-media/src/internals/GroupCoordinatorState.ts

```diff
@@ -42,7 +42,7 @@ import {
     IPlaybackTrackDataChangeEvent,
 } from "./GroupPlaybackTrackData";
 import { TelemetryEvents } from "./consts";
-import { waitUntilConnected } from "@microsoft/live-share/bin/internals";
+import { waitUntilConnected } from "@microsoft/live-share/bin/internals/utils";
```

Comment:
should we export waitUntilConnected? it has the hidden annotation already, same question for any resource referencing `bin` or `src`.

`src` one might be okay, because only used in unit tests after my changes